webhooks.md

```diff
@@ -2,19 +2,38 @@
 
 The Fidel API uses [webhooks](https://en.wikipedia.org/wiki/Webhook) to notify your application when relevant events happen in your account. You can utilise this functionality to receive data from events not triggered by direct API requests. Or to get the data in a service that is not responsible for making the API request but needs to consume the response.
 
-There are several webhooks available to use with the Fidel API, each corresponding to an event happening in your account: `brand.consent`, `card.linked `, `card.failed `, `program.status`, `location.status`, `transaction.auth`, `transaction.auth.qualified`, `transaction.clearing`, `transaction.clearing.qualified`, `transaction.refund` and `transaction.refund.qualified`. Each of them requires an HTTPS URL to be registered.
+There are several webhooks available to use with the Fidel API, each corresponding to an event happening either at **Program** or **Account** level. 
```

Comment:
"either at **program** or **account** level."

The objects (program, card, account, etc.) don't need to start with a capital letter.